### ISSUE TITLE
uwsgi: error while loading shared libraries: libicui18n.so.58: cannot open shared object

### ISSUE THREAD

**Issue by jrhayward** [conda-forge/uwsgi-feedstock#26]
Issue:
I get the uwsgi: error while loading shared libraries: libicui18n.so.58: cannot open shared object file: No such file or directory 
after using condo to install uwsgi in a docker container running ubuntu.  I tried the recommendation supplied here
https://github.com/conda-forge/uwsgi-feedstock/issues/7
but it did not resolve my issue.  

<br/>
(base) root@6039f3628b61:/# conda install uwsgi
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /root/miniconda3

  added / updated specs:
    - uwsgi

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    pcre-8.41                  |    hf484d3e_1003         249 KB  conda-forge
    ------------------------------------------------------------
                                           Total:         249 KB

The following NEW packages will be INSTALLED:

  jansson            conda-forge/linux-64::jansson-2.11-h516909a_1001
  pcre               conda-forge/linux-64::pcre-8.41-hf484d3e_1003
  uwsgi              conda-forge/linux-64::uwsgi-2.0.18-py37h4472670_1

Proceed ([y]/n)? y

Downloading and Extracting Packages
pcre-8.41            | 249 KB    | ##################################### | 100% 
Preparing transaction: done
Verifying transaction: done
Executing transaction: done

(base) root@6039f3628b61:/# uwsgi --version
uwsgi: error while loading shared libraries: libicui18n.so.58: cannot open shared object file: No such file or directory
(base) root@6039f3628b61:/# 

```
$ conda list

     active environment : base
    active env location : /root/miniconda3
            shell level : 1
       user config file : /root/.condarc
 populated config files : /root/.condarc
          conda version : 4.7.10
    conda-build version : not installed
         python version : 3.7.3.final.0
       virtual packages : 
       base environment : /root/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /root/miniconda3/pkgs
                          /root/.conda/pkgs
       envs directories : /root/miniconda3/envs
                          /root/.conda/envs
               platform : linux-64
             user-agent : conda/4.7.10 requests/2.22.0 CPython/3.7.3 Linux/4.9.125-linuxkit ubuntu/18.04.2 glibc/2.27
                UID:GID : 0:0
             netrc file : None
           offline mode : False
</details>

<br/>
Details about  <code>conda</code> DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.2 LTS"
NAME="Ubuntu"
VERSION="18.04.2 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.2 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

Docker 
Docker version 18.09.2, build 6247962
 ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

**Comment by jrhayward** [comment]
I have reduced this to this docker file and the problem persists
FROM ubuntu
RUN apt-get update
VOLUME test
COPY Miniconda3-latest-Linux-x86_64.sh  /
RUN bash Miniconda3-latest-Linux-x86_64.sh -b
ENV PATH /root/miniconda3/bin:/root/miniconda3/condabin:$PATH
RUN conda update conda
SHELL ["/bin/bash", "-c"]
RUN echo "source activate base" > ~/.bashrc
RUN conda init bash
RUN conda config --add channels conda-forge
RUN conda install -c conda-forge uwsgi
RUN conda update --all
RUN apt-get update && apt-get -y upgrade
VOLUME test

**Comment by ocefpaf** [comment]
If you are in a hurry you can do `conda install icu=58` to fix that for now.
A proper fix is underway... Hold on!

**Comment by jdhao** [comment]
@ocefpaf  I am still seeing this error. 
I am trying to create a env using the following command:

```
conda create -n py36 -c conda-forge python==3.6.5 uwsgi
```

After activating the env, when I use `uwsgi --version`, I am seeing the same error, though `conda install icu=58` solve the problem.

**Comment by ocefpaf** [comment]
> @ocefpaf I am still seeing this error.
> I am trying to create a env using the following command:
> 
> ```
> conda create -n py36 -c conda-forge python==3.6.5 uwsgi
> ```
> 
> After activating the env, when I use `uwsgi --version`, I am seeing the same error, though `conda install icu=58` solve the problem.

`conda-forge` dropped support for Python 3.6 before this was fixed, so it will work only on >=3.7.
